pkg/util/validation/validation.go

```diff
@@ -102,6 +102,16 @@ func ValidateSpreadConstraint(spreadConstraints []policyv1alpha1.SpreadConstrain
 			allErrs = append(allErrs, field.Invalid(fldPath.Index(index), constraint, "spreadByLabel should not co-exist with spreadByField"))
 		}
 
+		// If MinGroups provided, it should not be lower than 0.
+		if constraint.MinGroups < 0 {
+			allErrs = append(allErrs, field.Invalid(fldPath.Index(index), constraint, "minGroups lower than 0 is not allowed"))
+		}
+
+		// If MaxGroups provided, it should not be lower than 0.
+		if constraint.MaxGroups < 0 {
+			allErrs = append(allErrs, field.Invalid(fldPath.Index(index), constraint, "maxGroups lower than 0 is not allowed"))
+		}
+
```

Comment:
Do we need to check that `MaxGroups` cannot be less than `MinGroups`?

Reply:
Sorry, I missed it. There's already a judgment in the logic.

Reply:
`MaxGroups` could equal 0? If so, it means what? i.e., HPA minReplicas could only be >=1. I also think `MinGroups` should >=1

Reply:
No, after mutating, `MaxGroups` could be more than 0 or less than 0, never be 0.